index.js

```diff
@@ -1,8 +1,4 @@
-module.exports = function(str) {
-  if (!str) {
-    return null;
-  }
-
-  var d = new Date(String(str));
+module.exports = function(val) {
+  var d = new Date(val);
```

Comment:
@tjmehta  I think falsey inputs like `dateParse()` or `dateParse(null)` should return null instead of current date
Otherwise looks good